Tasks/UseRubyV1/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "use-ruby",
+  "version": "0.0.0",
+  "description": "Use Ruby Ecosystem",
+  "main": "useruby.js",
+  "scripts": {
+    "test": "mocha Tests/L0.js",
+    "build": "tsc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "keywords": [
+    "Azure",
+    "Pipelines",
+    "Tasks",
+    "Ruby"
+  ],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "azure-pipelines-task-lib": "^2.8.0",
```

Comment:
We should be using `3.0.0-preview`. Otherwise this will get tested on the old node handler, not the node 10 handler

Reply:
Done